### DISCUSSION TITLE
TS0601_dimmer

### DISCUSSION THREAD

**Discussion by alterco** [Koenkk/zigbee2mqtt#7608] [Devices]
Greetings. Tell me, can I set the minimum brightness for TS0601_dimmer?
Now, at a minimum, the lamp still glows and there is a voltage of 50V at the output of the dimmer. It is a defect or there is a factory setting with a non-zero minimum?

**Comment by infeeeee** [comment]
Have you found anything about this? 

I have a similar problem, but with the other way: The light only glows up at above ~40%. So I guess it's sending too low voltage for the bulb.

According to the tuya zigbee docs, there are zigbee codes for setting and reading the minimum and maximum brightness: 

https://developer.tuya.com/en/docs/iot-device-dev/tuya-smart-dimmer-switch-single-phrase-input-without-neutral-line?id=K9ik6zvokodvn#subtitle-10-Private%20cluster

From this link:

---
<html><body>
<h5 id="subtitle-10-Private%20cluster" class="comp-markdown-title">Private cluster</h5>
<p><font size="2" color="#ff"><strong>Attribute:</strong></font></p>


ID | Name | Data type | Range | Defualt value
-- | -- | -- | -- | --
0xFC00 | Level control max min | uint16 -0x21 | 0x0000 - 0xffff | 0x01ff
0xFC02 | Level control bulb type | uint8 -0x20 | 0x00–0xFF | 0x00
0xFC03 | Level control scr state | uint8 -0x20 | 0x00–0xFF | 0x01
0xFC04 | Level control current percentage | uint8 -0x20 | 0x00–0xFF | 0x01
0xFC05 | Level control min percentage | uint8 -0x20 | 0x00–0xFF | 0x01


</body>
</html>

---

But I don't know enough about zigbee to decipher this

**Reply by infeeeee** [reply]
For reference, these 2 dimmers support setting the minimum brightness via zigbee2mqtt:

Ubisys D1

https://www.zigbee2mqtt.io/devices/D1.html

Sinopé DM2500ZB

https://www.zigbee2mqtt.io/devices/DM2500ZB.html

Maybe worth taking a look about how they do it.

**Comment by ciB89** [comment]
Would also be interested in such a feature

**Comment by kevincaradant** [comment]
Hi 
Sorry to ask here but maybe that you know.

I would buy "Moes EDM-1ZBB-EU"  recognized as "TS0601_dimmer"

But what is the behavior after a power outage when power is back ? Restore previous state? On ? OFF ? 

Because I can't see any way to change it via Z2M. I'm currently hesitate to buy it for that :)

Thanks
